skyrim-platform/src/tilted/core_library/Filesystem.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Filesystem.hpp>
```

Comment:
Wow. Now when I look at it, I see a lot of deleted and then added files unrelated to this PR. Maybe it would be better to make a separate PR with just renaming `third_party/cef` to `tilted`? History would be a lot cleaner